pages/_app.js

```diff
@@ -123,12 +126,13 @@ class OpenCollectiveFrontendApp extends App {
     const { client, Component, pageProps, scripts, locale, messages } = this.props;
 
     const intl = createIntl({ locale: locale || 'en', messages }, cache);
+    const stripePromise = loadStripe(getEnvVar('STRIPE_KEY'));
 
     return (
       <Fragment>
         <ApolloProvider client={client}>
           <ThemeProvider theme={theme}>
-            <StripeProviderSSR>
+            <Elements stripe={stripePromise}>
```

Comment:
We don't want to load stripe on all pages, that's why we had `StripeProvider`/`loadStripe`.

Reply:
Addressed in https://github.com/opencollective/opencollective-frontend/pull/6193 😄